src/quo/components/tags/context_tag/schema.cljs

```diff
@@ -0,0 +1,103 @@
+(ns quo.components.tags.context-tag.schema
+  (:require [malli.core :as malli]))
+
+(def ^:private ?base
+  [:map
+   [:type {:optional true}
+    [:maybe
+     [:enum :default :multiuser :group :channel :community :token :network :multinetwork :account
+      :collectible :address :icon :audio]]]
+   [:customization-color {:optional true} [:maybe :schema.common/customization-color]]
+   [:theme :schema.common/theme]
+   [:blur? {:optional true} [:maybe :boolean]]
+   [:state {:optional true} [:maybe [:enum :selected :default]]]])
+
+(def ^:private ?size
+  [:map
+   [:size {:optional true} [:maybe [:enum 24 32]]]])
+
+(def ^:private ?default
```

Comment:
The name `?default` seems to conflict with `?base`. Both feel like they do the same thing, but they are actually different. 

How about calling this something else ? Like `?profile-base` ?

Reply:
to avoid the name conflict we could merge the `?base` map with the `:multi` schema. This way we won't have to repeat it everywhere also.

Reply:
I renamed it to `?context-base`